### PR TITLE
i18n: add structure for fr

### PR DESCRIPTION
**Pull request by Bredda**
Hi,

I signed the CLA.

Modifications are for french translation
- Add structure for french translation
- Translation done for:
   - Sidenav titles
   - Contributing
   - main index
   - "Getting started"  chapter

Best regards,
Laurent

### CODE REVIEW

File: docs_source_files/config.toml [hunk 1]
```diff
@@ -136,3 +142,17 @@ weight = 20
 name = "<i class='fas fa-fw fa-bullhorn'></i> How to contribute"
 url = "contributing"
 weight = 30
+[[Languages.fr.menu.shortcuts]]
```
Comment (diemol) [inline docs_source_files/config.toml:145]: @Bredda could you please check the config file? Seems the `zh-cn` configuration and the `fr` one are overlapping. Please also check the output of `hugo server`, seems some links are broken.

**Review by diemol — CHANGES_REQUESTED** [review]

File: docs_source_files/config.toml (hunk 1 above)
Comment (Bredda) [reply docs_source_files/config.toml:145]: My bad. Fixed both conf file and links.
I should be ok now.

Comment (diemol) [reply docs_source_files/config.toml:145]: Thanks!

**Review by diemol — APPROVED** [review]
Thank you @Bredda, good job!

**Pull request merged**